src/kakarot/accounts/uninitialized_account.cairo

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.alloc import alloc
+from starkware.starknet.common.syscalls import get_caller_address, replace_class, library_call
+
+from kakarot.accounts.shared_storage import Account_evm_address, Account_kakarot_address
```

Comment:
I'm hesitating because any change to this file would invalidate the deterministic evm to starknet address. I'm not even sure if just a renaming wouldn't make it as well

Reply:
we need to make sure by compiling and computing class hash in different cases,

Reply:
Duplicated storage vars in both files instead.